docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -266,31 +271,30 @@ You can use `quarkus.keycloak.devservices.realm-path` to initialize Keycloak wit
 
 == Disable Dev Services for Keycloak
 
-`Dev Services For Keycloak` will not be activated if either `quarkus.oidc.auth-server-url` is already initialized or the default OIDC tenant is disabled with `quarkus.oidc.tenant.enabled=false`, irrespectively of whether you work with Keycloak or not.
+Dev Services for Keycloak is not activated if either `quarkus.oidc.auth-server-url` is already initialized or the default OIDC tenant is disabled with `quarkus.oidc.tenant.enabled=false`, regardless of whether you work with Keycloak or not.
 
-If you prefer not to have a `Dev Services for Keycloak` container started or do not work with Keycloak then you can also disable this feature with `quarkus.keycloak.devservices.enabled=false` - it will only be necessary if you expect to start `quarkus:dev` without `quarkus.oidc.auth-server-url`.
+If you prefer not to have a Dev Services for Keycloak container started or do not work with Keycloak, then you can also disable this feature with `quarkus.keycloak.devservices.enabled=false` - it is only necessary if you expect to start `quarkus:dev` without `quarkus.oidc.auth-server-url`.
 
-The main Dev UI page will include an empty `OpenID Connect Card` when `Dev Services for Keycloak` is disabled and the `quarkus.oidc.auth-server-url` property
-has not been initialized:
+The main Dev UI page includes an empty *OpenID Connect* card when Dev Services for Keycloak is disabled and the `quarkus.oidc.auth-server-url` property has not been initialized:
 
-image::dev-ui-oidc-card.png[alt=Dev UI OpenID Connect Card,role="center"]
+image::dev-ui-oidc-card.png[alt=Dev UI OpenID Connect card,role="center"]
 
-If `quarkus.oidc.auth-server-url` is already set then a generic OpenID Connect Dev Console which can be used with all OpenID Connect providers may be activated, please see <<dev-ui-all-oidc-providers,Dev UI for all OpenID Connect Providers>> for more information.
+If `quarkus.oidc.auth-server-url` is already set, then a generic OpenID Connect Dev Console, which can be used with all OpenID Connect providers, can be activated. For more information, see the <<dev-ui-all-oidc-providers,Dev UI for all OpenID Connect Providers>> section.
 
 [[dev-ui-all-oidc-providers]]
-== Dev UI for all OpenID Connect Providers
+== Dev UI for all OpenID Connect providers
 
-If `quarkus.oidc.auth-server-url` points to an already started OpenID Connect provider (which can be Keycloak or other provider), `quarkus.oidc.auth-server-url` is set to `service` (which is a default value) or `hybrid` and at least `quarkus.oidc.client-id` is set then `Dev UI for all OpenID Connect Providers` will be activated.
+If `quarkus.oidc.auth-server-url` points to an already started OpenID Connect provider (which can be Keycloak or other provider), `quarkus.oidc.auth-server-url` is set to `service` (which is a default value) or `hybrid` and at least `quarkus.oidc.client-id` is set then `Dev UI for all OpenID Connect Providers` is activated.
```

Comment:
Good suggestion, @sheilamjones. I took it one step further with:
```suggestion
The Dev UI for all OpenID Connect Providers is activated if the following conditions are met:

* The `quarkus.oidc.auth-server-url` points to an already started OpenID Connect provider, which can be Keycloak or other provider.
* The `quarkus.oidc.auth-server-url` is set to `service`, the default value, or `hybrid`.
* The `quarkus.oidc.client-id` is set.
```

Reply:
Hi @rolfedh,  
Your update looks great and really clear. I just noticed two very minor things that perhaps could be changed:
- Change capitalization "P" in "OpenID Connect Providers" to "providers"
- Propose to say "which can be Keycloak or another provider"  (update to say "another" instead of "other"